docs/asyncapi.yaml

```diff
@@ -37,171 +39,120 @@ components:
                 maximum: 9999999999
                 example: 1234567890
               apiLink:
-                type: url
                 description: api link for calling file stream of statement file that has been published unto blob
                 example: 'https://myStatementRetreivalApiEndpoint/statement-receiver/statement/v1/FFC_PaymentStatement_SFI_2022_1234567890_2022090615023001.pdf'  
               documentType:
                 type: string
                 enum:
                   - Payment statement
+                  - Payment schedule
                 description: document type
-                example: 'Payment statement'
+                example: 
+                  items:
+                    - Payment statement
+                    - Payment schedule
               scheme:
                 type: string
                 enum:
                   - SFI
                 description: Scheme name
                 example: SFI
-          type: 'uk.gov.pay.statement.crm'
-          source: 'ffc-pay-statement-generator'
   schemas:
     PaymentStatement:
+      additionalProperties: false
       type: object
       required:
       - businessName
-      - sbi
       - frn
+      - sbi
       - address
-      - scheme
-      - payments
       - funding
+      - fundingTotal
```

Comment:
This property isn't in the object

Reply:
Should I just remove this reference to it?

Reply:
Yup